server/routes/order.router.js

```diff
@@ -0,0 +1,136 @@
+const express = require('express');
+const pool = require('../modules/pool');
+const router = express.Router();
+const {
+  rejectUnauthenticated,
+} = require('../modules/authentication-middleware');
+
+/**
+ * GET route template
+ */
+router.get('/', rejectUnauthenticated, async (req, res) => {
+  try {
+    const queryText = `
+    SELECT * FROM "orders" 
+    WHERE orders."companyID" = $1 
+    ORDER BY ("testingStatus" = 'Pre-shipment') DESC;`;
+    const dbRes = await pool.query(queryText, [req.user.companyID]);
+    res.send(dbRes.rows);
+  } catch (err) {
+    console.error(err.message);
+    res.sendStatus(500);
+  }
+});
+
+router.put('/newOrder/:id', async (req, res) => {
+  try {
+    const userId = req.user.id;
+    const shipping = req.body;
+    const sqlParams = [
+      shipping.id,
+      userId,
+      shipping.shippedDate,
+      shipping.carrierName,
+      shipping.trackingNumber,
+    ];
+    const sqlQuery = `UPDATE "orders" 
+    SET "shippedDate" = $3, "carrierName" = $4, "trackingNumber" = $5
+    WHERE "id" = $1 
+    RETURNING "id";`;
+    const dbRes = await pool.query(sqlQuery, sqlParams)
+    res.send(dbRes.rows[0])
```

Comment:
Consider adding:

```js
if (dbRes.rows.length === 0) {
   res.sendStatus(404);
   return;
}
```

Otherwise, your server will send back `undefined` for a bad ID, and make it harder to track down the bug.